projects/plugins/super-cache/changelog/init-release-cycle#2

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: changed
+Comment: Init 1.12.0-alpha
```

Comment:
I always get confused with versioning. Isn't it suppose to be 1.11.1-alpha?

Reply:
I think so too. Based on the autoloader suffix and package.json

Reply:
You're right. I changed it elsewhere but I forgot that one.

Reply:
Fixed it!